frontend/ui/widget/numberpickerwidget.lua

```diff
@@ -109,9 +109,7 @@ function NumberPickerWidget:init()
         end
     }
 
-    local empty_space = VerticalSpan:new{
-        width = math.ceil(self.screen_height * 0.01)
-    }
+    local empty_space = VerticalSpan:new{ width = Size.span.horizontal_default }
```

Comment:
~~Feel free to add a Size.span.vertical_verylarge, using a horizontal_something for a VerticalSpan feels a bit odd :) (I'm pretty sure there is such code because one was lazy, possibly me :), but we may at least try to do it right :)
Feels odd we are already using a lot of vertical_large = 5 only, which doesn't look very large... Should have better be renamed in all the places to vertical_default or medium...)~~

Or just use Size.padding.lage which has the same value as your Size.span.horizontal_default : you use it for padding, so that will read well :)

Reply:
Done